gunrock/app/mf/mf_app.cu

```diff
@@ -282,31 +288,28 @@ float mf(
     parameters.Parse_CommandLine(0, NULL);
     parameters.Set("graph-type", "by-pass");
     parameters.Set("num-runs", num_runs);
-    // TODO: problem specific inputs, e.g.:
-    std::vector<VertexT> srcs;
-    for (int i = 0; i < num_runs; i ++)
-        srcs.push_back(sources[i]);
-    parameters.Set("srcs", srcs);
+    parameters.Set("source", source);
+    parameters.Set("sink", sink);
 
     bool quiet = parameters.Get<bool>("quiet");
-    GraphT graph;
+    CsrT csr;
     // Assign pointers into gunrock graph format
-    // TODO: change to other graph representation, if not using CSR
-    graph.CsrT::Allocate(num_nodes, num_edges, gunrock::util::HOST);
-    graph.CsrT::row_offsets   .SetPointer(row_offsets, gunrock::util::HOST);
-    graph.CsrT::column_indices.SetPointer(col_indices, gunrock::util::HOST);
-    graph.CsrT::edge_values   .SetPointer(edge_values, gunrock::util::HOST);
-    graph.FromCsr(graph.csr(), true, quiet);
+    csr.Allocate(num_nodes, num_edges, gunrock::util::HOST);
+    csr.row_offsets   .SetPointer(row_offsets,gunrock::util::HOST);
+    csr.column_indices.SetPointer(col_indices,gunrock::util::HOST);
+    csr.capacity      .SetPointer(capacity,   gunrock::util::HOST);
```

Comment:
These `.SetPointer` commands are missing the `size` parameter.
Line# 298, 299, 300 needs fixing, see: https://github.com/gunrock/gunrock/blob/dev-refactor/gunrock/app/pr/pr_app.cu#L347

Reply:
Do I need that function if my algorithm use only the CSR graph representation?

Reply:
Yes, you still have to set the right pointers of the graph to right arrays that you get from the c or python interface. You can skip this entire function and the one above it called `gunrock_mf` if you don't want to implement the c and python bindings. But eventually, we will need these.